web/pdf_viewer.js

```diff
@@ -295,7 +295,6 @@ var PDFViewer = (function pdfViewer() {
       var onePageRendered = new Promise(function (resolve) {
         resolveOnePageRendered = resolve;
       });
-      this.onePageRendered = onePageRendered;
```

Comment:
I'm not sure I agree with removing this one completely, since there might still be cases where it could be useful. Note that this property is currently available in e.g. viewer components.